web/src/main/java/org/fao/geonet/EncryptorInitializer.java

```diff
@@ -239,7 +246,10 @@ private void updateDb(StandardPBEStringEncryptor previousEncryptor) throws SQLEx
             }
 
             for (Integer key : updatesMapServers.keySet()) {
-                statement.execute("UPDATE Mapservers SET password='" + updatesMapServers.get(key) + "' WHERE id='" + key + "'");
+                PreparedStatement pstmt = connection.prepareStatement("UPDATE Mapservers SET password==? WHERE id=?");
```

Comment:
```suggestion
                PreparedStatement pstmt = connection.prepareStatement("UPDATE Mapservers SET password=? WHERE id=?");
```
?

Reply:
Thanks @fxprunayre, apologies as I missed it in the review. Going to do a PR to fix it.